### ISSUE TITLE
Side-by-side list items

### ISSUE THREAD

**Issue by xicreative** [PolymerElements/iron-list#105]
I was wondering if there's a way to let list items be side-by-side. I have a grid-layout view that would be fantastic with this element, but I haven't been able to figure out a way to let my elements stack side-by-side.

example:
```html
<style>
  custom-tile {
     width: 50%;
  }
</style>

<main class="fit layout vertical">
  <nav>...</nav>
  <custom-gallery class="layout horizontal wrap">
    <iron-list items="[[data]]" as="item">
       <template>
          <custom-tile item="[[item]]"></custom-tile>
       </template>
    </iron-list>
  </custom-gallery>
</main>
```

I'd expect it to have two `<custom-tile>` elements side-by-side, but because of 
```css
#index > ::content > * {
  width: 100%;
}
```
 I get weirdness. Any suggestions? Is `#index` a necessary element at all?

**Comment by hawkett** [comment]
There's a horizontal mode [on the roadmap](https://github.com/PolymerElements/iron-list/issues/10), though not sure if the intention with that is specifically what your looking for.

**Comment by davidrissato** [comment]
I made a "quick and dirt" solution to achieve such grid-like behavior. https://easyusedev.wordpress.com/2015/10/02/polymer-how-to-iron-list-horizontal-grid/

**Comment by blasten** [comment]
This feature is the backlog, and it will be implemented at some point.

**Comment by blasten** [comment]
This feature was released in v1.3.0: https://github.com/PolymerElements/iron-list/releases/tag/v1.3.0
